src/main/java/it/pagopa/ecommerce/commons/client/NpgClient.java

```diff
@@ -839,8 +686,15 @@ private NpgResponseException exceptionToNpgResponseException(
                     default -> List.of();
                 };
 
-                errors = responseErrors.stream()
-                        .map(error -> GatewayError.valueOf(error.getCode())).toList();
+                errors = Optional
+                        .ofNullable(responseErrors)
+                        .map(
+                                errorDetails -> errorDetails
+                                        .stream()
+                                        .map(ErrorsInnerDto::getCode)
+                                        .toList()
+                        )
+                        .orElse(List.of());
```

Comment:
Easier to read maybe?

```suggestion
                errors = Optional
                        .ofNullable(responseErrors)
                        .orElse(List.of())
                        .stream()
                        .map(ErrorsInnerDto::getCode)
                        .toList();
```

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/178/commits/000cab563e84831ac0eef2ca29fde9df363f7095